apps/omg_performance/lib/performance.ex

```diff
@@ -93,12 +91,6 @@ defmodule OMG.Performance do
     :ok = Application.put_env(:ethereumex, :request_timeout, :infinity)
     :ok = Application.put_env(:ethereumex, :http_options, recv_timeout: :infinity)
     :ok = Application.put_env(:ethereumex, :url, opts[:ethereum_rpc_url])
-
-    :ok =
-      if opts[:contract_addr],
-        do: Application.put_env(:omg_eth, :contract_addr, contract_map_to_hex(opts[:contract_addr])),
```

Comment:
I have no idea what was going on here... SetContracts release task takes care of this now